snap/README.md

```diff
@@ -0,0 +1,99 @@
+* [Kata Containers snap image](#kata-containers-snap-image)
+* [Initial setup](#initial-setup)
+* [Build snap image](#build-snap-image)
+* [Install snap (developer)](#install-snap-developer)
+* [Integration with docker](#integration-with-docker)
+* [Limitations](#limitations)
+
+# Kata Containers snap image
+
+This directory contains the resources needed to build the Kata Containers
+[snap][1] image.
+
+## Initial setup
+
+*Ubuntu 18 (Bionic Beaver)*
```

Comment:
Nit: could you make that "18.04" as "even major number" + "04" minor number == LTS :) Plus when 18.10 comes out, this will be wrong as it won't be Bionic any more ;)